src/Integration/Language.cs

```diff
@@ -57,16 +58,24 @@ public static IEnumerable<Language> KnownLanguages
             }
         }
 
-        public Language(string serverKey, string name, string projectTypeGuid)
+        private Language(string name)
         {
-            if (string.IsNullOrWhiteSpace(serverKey))
+            if (string.IsNullOrWhiteSpace(name))
             {
```

Comment:
Why is this private constructor needed? seems like it could be merged with the public one.